spec/lib/manageiq/api/common/headers_spec.rb

```diff
@@ -10,4 +10,12 @@
       expect { ManageIQ::API::Common::Headers.current = header_bad }.to raise_exception(ArgumentError)
     end
   end
+
+  context "get forwardable headers" do
+    it "x-rh-identity" do
+      ManageIQ::API::Common::Headers.current = header_good
+      hash = ManageIQ::API::Common::Headers.forwardable_headers
```

Comment:
Minor, but you can use `described_class` instead of typing out the entire class name.  Useful for if it ever gets renamed.